### PR TITLE
Feature/mobile version

### PR DESCRIPTION
**Pull request by SirriCelles**
- Create the 2 pages for mobile.
- Adapt the 2 pages to desktop.
- Implement the user interactions: link, mobile menu.
- Implement the section "Featured speakers" with dynamic HTML
- Deploy the project and check to make sure everything works as planned.
-  Create a Vidoe describing the project. You can find the link to my video [Here](https://www.loom.com/share/c7546d65c129498f8cafca6645c5d81c)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+              <a href="./index.html" class="nav__logo">
+                <img
+                  src="./assets/img/Women-in-Tech-Logo.png"
+                  alt="Women in tech logo"
+                />
+              </a>
+              <div class="nav__items">
+                <ul class="nav__list">
+                  <li class="nav__list-items">
+                    <a href="./about.html" class="nav__list-link">About</a>
```
Comment (prolajumokeoni) [inline about.html]: - [x] Kindly change this link from about.html to index.html to users can easily navigate back to the home page from the about page 👍🏿

File: about.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-microsoft"></use>
+          </svg>
+        </div>
+      </div>
+
+      <footer class="footer" id="about-footer">
+        <div class="legal">
+          &copy;2022. All rights reserved.
+      </div>
+      </footer>
```
Comment (prolajumokeoni) [inline about.html]: - [x] Your footer should contain a logo aligned to the left and text aligned to the right. You can also add this footer to your index.html page even though the main design does not have a footer in the index.html page 👍🏿

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,632 @@
...
+html {
+  font-size: 62.5%; }
+
+body {
+  background-color: var(--color-background-dark);
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.6rem;
+  line-height: 2.4rem; }
+  @media (max-width: 767px) {
```
Comment (prolajumokeoni) [inline css/style.css:32]: - [x]  Your  CSS styling should be built as **mobile-first**,  please don't use max-width in your mobile design, don't use media queries in your mobile version, kindly remove all media queries in your mobile version 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @SirriCelles, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [x] Please use the [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter, as this is part of the project requirement 👍🏿 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/programs.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    count = 2;
+  }
+
+  for(let i=0; i < count; i++) {
+    let item = document.createElement('div');
+    item.className = "speaker__wrapper";
+    const newSpeaker = `<div class="speaker__image">
+    <img src="${speakers[i].imageIcon}" alt="Chess icon" class="speaker__image-icon">
+      <img src="${speakers[i].imageUser}" alt="speaker" class="speaker__image-user">
+    </div>
```
Comment (prolajumokeoni) [inline js/programs.js]: - Good job working on this feature 💪🏿 😎

File: README.md [hunk 5]
```diff
@@ -1,2 +1,69 @@
...
+
+- GitHub: [@SirriCelles](https://github.com/SirriCelles)
+- Twitter: [@SirriCelles](https://twitter.com/SirriCelles?t=fZl0blItFUQDC5vozH47nA&s=09)
+- LinkedIn: [Sirri Celles](https://www.linkedin.com/in/sirri-celles)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (prolajumokeoni) [inline README.md]: - [x] Kindly fix this broken link, this will make your readme more professional 👍🏿

File: css/style.css (hunk 3 above)
Comment (SirriCelles) [reply css/style.css:32]: @prolajumokeoni  Thank you for the detailed review.
Please could you explain what you mean? I have just one CSS file for both mobile and desktop

Comment (SirriCelles) [reply css/style.css:32]: @prolajumokeoni  After some thinking, I understand now. I have implemented the above changes

**Review by alexander16108 — APPROVED** [review]
Hi @SirriCelles , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**